components/drip/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "@pipedream/drip",
+  "version": "0.0.1",
+  "description": "Pipedream Drip Components",
+  "main": "drip.app.mjs",
+  "keywords": [
+    "pipedream",
+    "drip"
+  ],
+  "homepage": "https://pipedream.com/apps/drip",
+  "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+  "license": "MIT",
+  "dependencies": {
+    "@pipedream/platform": "^0.10.0"
```

```suggestion
    "@pipedream/platform": "^1.2.1"
```
How about using the latest version?